web/src/poll_modal.ts

```diff
@@ -1,15 +1,15 @@
 import $ from "jquery";
-import {Sortable} from "sortablejs";
+import SortableJS from "sortablejs";
 
 import render_poll_modal_option from "../templates/poll_modal_option.hbs";
 
-function create_option_row($last_option_row_input) {
+function create_option_row($last_option_row_input: JQuery<HTMLInputElement>): void {
     const row = render_poll_modal_option();
     const $row_container = $last_option_row_input.closest(".simplebar-content");
     $row_container.append(row);
 }
 
-function add_option_row(e) {
+function add_option_row(e: {target: JQuery<HTMLInputElement>}): void {
```

Comment:
You should type `e` properly, rather than one of its fields; you can use types like `JQuery.ClickEvent`. `git grep "e: J"` will find you examples.